.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Product Release Checks
+about: Use this template to track all release checks for your component
+title: "[Product] Release Checks"
+labels: compliance, documentation, "security analysis", "test results"
```

Comment:
Maybe we can also discuss further to reduce the amount of labels for release-management maybe only stay with "release-management" so that is clear that theese issue templates are only maintained and used by RM 

but could also imagine that filtering between the sections would be a bit harder... but IMHO we should go for that.

Reply:
That's true. Just added all labels, that have been added to the templates I deleted. 
If that's ok with you, than I would discuss that outside of this PR